data_validation/__main__.py

```diff
@@ -159,13 +160,24 @@ def get_calculated_config(args, config_manager):
     return calculated_configs
 
 
-def build_config_from_args(args, config_manager):
-    """Return config manager object ready to execute.
+def build_config_from_args(args: Namespace, config_manager: ConfigManager):
+    """Append build configs to ConfigManager object.
 
     Args:
+        args (Namespace): User specified Arguments
         config_manager (ConfigManager): Validation config manager instance.
     """
+    # Append SCHEMA_VALIDATION configs
+    if config_manager.validation_type == consts.SCHEMA_VALIDATION:
```

Comment:
We don't need to add a condition for schema validation since this method only gets called for non-schema validations: https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/__main__.py#L304

Reply:
I have put all the post build configs under `build_config_from_args(args, config_manager)`: https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/issue673-optimize-cli-tools-arg-parser/data_validation/__main__.py#L257

Hence all the 4 types of validations are checked and compared here including schema-validations